NetflixRouletteWebpack/src/components/modals/edit-movie-modal/EditMovieModal.js

```diff
@@ -63,7 +63,11 @@ function EditMovieModal({
       }
     },
     onReset: () => {
-      setSelectedGenres([]);
+      if (modalTitle === "Add movie") {
```

Comment:
@Marysosh Can be simplified like `setSelectedGenres(modalTitle === "Add movie" ? "" : initialValues.genre);`

Reply:
Yes, you're right, it looks better. Corrected✅